src/switch_apr.c

```diff
@@ -787,7 +787,7 @@ SWITCH_DECLARE(switch_status_t) switch_socket_send_nonblock(switch_socket_t *soc
 SWITCH_DECLARE(switch_status_t) switch_socket_sendto(switch_socket_t *sock, switch_sockaddr_t *where, int32_t flags, const char *buf,
 													 switch_size_t *len)
 {
-	if (!where || !buf || !len || !*len) {
+	if (!sock || !where || !buf || !len || !*len) {
```

Comment:
If I check the stack trace of these ticket. The socket was already validated before calling switch_socket_sendto
```
	if (!rtp_session || !rtp_session->sock_output) {
		return status;
	}

	switch_socket_sendto(rtp_session->sock_output, rtp_session->remote_addr, 0, rtp_packet, &len);
```
`sock_output` is not a NULL value but it can be a garbage pointer. Try to look where the sock_output is destroyed or deleted. I think what happened here is a race condition when FS destroy the socket and the zrtp callback is triggered